### PR TITLE
Fix Issue 3394

### PR DESCRIPTION
**Pull request by ltkum**
This will sanitize inputs and remove the need for accents in searches while keeping it possible to do so. 
in Issue #3394 , 
case with param_2 work, but param_3 still does not. The issue is different in this case, as it seems we are not unquoting those parameters server side. 

The error message I receive back is : 
```
'{"status":"error","code":400,"detail":"Cannot parse \'layerDefs\' %7B%22ch.swisstopo.amtliches-strassenverzeichnis%22%3A%20%22label%20like%20%27%25Controle%25%27%22%7D"}'
```
and if you decode the url (for example with https://meyerweb.com/eric/tools/dencoder/), I can tell that this is a correct URL encoding. so, I'll be investigating this now.
- [x] add a test for queries with accents
- [x] investigate url encoding problem
- [x] give a testlink in the issue

### CODE REVIEW

**Comment by ltkum** [comment]
@eraymann 
https://mf-chsdi3.dev.bgdi.ch/ltkum_dat_issue/ if you want to run some tests.

**Comment by eraymann** [comment]
It's working now, thank you very much. One remark: It's a small disadvantage that accents are not included when searching with layerDefs. But that's not too bad, because they are considered in the conventional search with searchText. I can live with it.

File: requirements.txt [hunk 1]
```diff
@@ -59,3 +59,4 @@ WebOb==1.8.5
 WebTest==2.0.33
 zope.deprecation==4.4.0
 zope.interface==4.6.0
+unidecode
```
Comment (procrastinatio) [inline requirements.txt]: Please, pin the version (1.1.1)

Comment (procrastinatio) [reply requirements.txt]: Nice

**Comment by procrastinatio** [comment]
@eraymann
Deploying the last version
https://mf-chsdi3.dev.bgdi.ch/ltkum_dat_issue/

**Pull request merged**